practice/timezone/timezone/views.py

```diff
@@ -0,0 +1,81 @@
+"""timezone Views."""
+
+from . import logic
+import arrow
+
+from django.http import HttpResponse
+
+
+def get_qualified_tz(latlng_string):
+    r"""Check the requested lat, lng for validity.
+
+    >>> get_qualified_tz('45,45')
+    'Europe/Moscow'
+    >>> get_qualified_tz('4545')
+    Traceback (most recent call last):
+    ...
+    KeyError: 'Need two values'
+    >>> get_qualified_tz('45,-125')
+    Traceback (most recent call last):
+    ...
+    OSError: Time Zone does not exist there.
+    """
+    lat_lng_list = latlng_string.split(',')
+    if len(lat_lng_list) != 2:
+        raise KeyError('Need two values')
+    tz_at_latlng = logic.get_tz(lat_lng_list)
+    if tz_at_latlng is None:
+        raise OSError('Time Zone does not exist there.')
```

Comment:
I think both of these exceptions should be ValueErrors which represent an error upon receiving invalid input. Look at the documentation for the various exception types: https://docs.python.org/3.5/library/exceptions.html#concrete-exceptions

KeyErrors represent a dictionary lookup failed. OSErrors represent something going wrong with a lower-level operating system call. Using these errors here make the code harder to understand since it implies some interfacing with dictionaries and the OS, which it doesn't have.